projects/wikimedia.org.yaml

```diff
@@ -8,33 +8,44 @@ paths:
             title: Wikimedia REST API
             description: >
                 This API provides cacheable and straightforward access to
-                Wikimedia content and data, in machine-readable formats. Each
-                entry point has explicit stability markers to inform you about
-                development status and change policy, according to [our API
-                version policy](https://www.mediawiki.org/wiki/API_versioning).
+                Wikimedia content and data, in machine-readable formats.
```

Comment:
Don't we need to add " in machine-readable formats, for the project specified in the URL." according to zhou's comments?